lib/active_model/serializer/associations.rb

```diff
@@ -13,7 +13,7 @@ module Associations
       included do
         with_options instance_writer: false, instance_reader: true do |serializer|
           serializer.class_attribute :_reflections
-          self._reflections ||= []
+          self._reflections ||= {}
```

Comment:
I can run my aeonvera project's tests with this branch/master, if you think that'll be reassuring.